### PR TITLE
NCDA: Nutrition measurements at NCDA form

### PR DESCRIPTION
**Pull request by anvmn**
#904

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2023-10-09 18-07

**Comment by anvmn** [comment]
Peek 2023-10-09 18-26

File: client/src/elm/Pages/WellChild/ProgressReport/View.elm [hunk 1]
```diff
@@ -2675,13 +2701,75 @@ viewFillTheBlanksPane language currentDate zscores child db groupNutritionMeasur
...
+                )
+                ncdaValuesSet
+                |> distributeByAgeInMonthsWithDate child
 
         weightsByAgeInMonths =
+            case ( weightsByAgeInMonthsWithDateFromNutrition, weightsByAgeInMonthsWithDateFromNCDA ) of
+                ( Just fromNutrition, Just fromNCDA ) ->
+                    Dict.merge
+                        (\key value -> Dict.insert key (Tuple.second value))
+                        -- In case we got both values for month, we give preference to
```
Comment (AronNovak) [inline client/src/elm/Pages/WellChild/ProgressReport/View.elm]: May we reduce code redundancy with https://github.com/TIP-Global-Health/eheza-app/pull/905/files#diff-53236e6f07f593ca22091310ef666c976e601040442e42239c653b9bc73ef6a2R2673 ?

Comment (anvmn) [reply client/src/elm/Pages/WellChild/ProgressReport/View.elm]: @AronNovak done

**Review by AronNovak — APPROVED** [review]
:+1:

**Comment by AronNovak** [comment]
I restarted the failed build.

**Pull request merged**